conf/supervisord.conf.jnj

```diff
@@ -170,6 +170,14 @@ autostart = {{ config['teamsyncworker']['autostart'] }}
 stdout_events_enabled = true
 stderr_events_enabled = true
 
+[program:repomirrorworker]
```

Comment:
Might be related to the missing worker comment ... but why was this added? Copy+Paste job?

Reply:
Bad merge after rebase